.github/actions/test/action.yml

```
@@ -32,7 +32,7 @@ runs:
       shell: bash
       run: |
         yarn clean
-        yarn gen
+        time yarn cross-env BUILD_THREADS=4 yarn gen
```

```suggestion
        ${{ runner.os != 'Windows' && 'time ' || '' }} yarn cross-env BUILD_THREADS=4 yarn gen
```

Here's a fix if you want to save the `time` command.